functional-tests/drand_test.go

```diff
@@ -0,0 +1,57 @@
+package functional
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/filecoin-project/go-filecoin/internal/app/go-filecoin/node"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/clock"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/drand"
+	tf "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/testflags"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDrandPublic(t *testing.T) {
+	tf.FunctionalTest(t)
+	t.Skip(("requires local drand setup"))
```

Comment:
I'm fine with this for now.  I've been advocating for having functional tests hit the default (i.e. public drand network) but maybe that's not a good idea.  cc @anorth who I've talked to about this.

Reply:
Yeah, I played around a bit with getting this to work against the existing public network, but it's just too far out of date.